pkg/kube/roundtripper.go

```diff
@@ -58,10 +58,10 @@ func (rt *RetryingRoundTripper) roundTrip(req *http.Request, retry int, prevResp
 	r.Seek(0, io.SeekStart)
 	resp.Body = io.NopCloser(r)
 	if err != nil {
-		return resp, rtErr
+		return resp, err
```

Comment:
I think this is waiting for more data, so it should return nil

Reply:
Modified. Please review again.